packages/strapi-generate-new/lib/utils/merge-template.js

```diff
@@ -169,36 +171,66 @@ async function checkTemplateContentsStructure(templateContentsPath) {
   checkPathContents(templateContentsPath, []);
 }
 
-function getRepoInfo(template) {
-  try {
-    const { user, project, default: urlStrategy } = gitInfo.fromUrl(template);
-    if (urlStrategy === 'https' || urlStrategy === 'http') {
-      // A full GitHub URL was provided, return username and project directly
-      return { user, project };
-    }
-    if (urlStrategy === 'shortcut') {
-      // A shorthand was provided, so prefix the project name with "strapi-template-"
-      return {
-        user,
-        project: `strapi-template-${project}`,
-      };
-    }
-  } catch (error) {
-    // If it's not a GitHub URL, then assume it's a shorthand for an official template
-    return {
-      user: 'strapi',
-      project: `strapi-template-${template}`,
-    };
+function parseShorthand(template) {
```

Comment:
can you move the logic into a fetch-github file like in the other CLI ?

Reply:
Sure, it's done